src/main/java/com/github/fge/jsonschema/core/load/configuration/LoadingConfigurationBuilder.java

```diff
@@ -137,28 +132,13 @@
         dereferencing = cfg.dereferencing;
         preloadedSchemas = Maps.newHashMap(cfg.preloadedSchemas);
         parserFeatures = EnumSet.copyOf(cfg.parserFeatures);
-        enableCache = cfg.enableCache;
         cacheSize = cfg.cacheSize;
     }
 
     /**
-     * Should we enable caching of downloaded schemas
-     *
-     * <p>Note that this does <b>not</b> affect preloaded schemas</p>
-     * 
-     * @param enableCache if loaded schemas have to be cached
-     * @return this
-     */
-    public LoadingConfigurationBuilder setEnableCache(final boolean enableCache)
-    {
-        this.enableCache = enableCache;
-        return this;
-    }
-
-    /**
      * How many schemas should be cached
-     * <p>Note if enableCache is false this setting is ignored</p>
-     * <p>Note setting enableCache to false or this to zero both effectively disable the cache</p>
+     * <p>Note setting to zero effectively disable the cache</p>
+     * <p>Note not settting the chache size or setting to -1, creates an unlimited cache</p>
```

Comment:
If you want to make this true don't you want to init cacheSize to -1 instead of 4096?  To be honest I'd prefer it to default to a fixed number instead of unlimited.  The unlimited can easily lead to "memory leaks".  If you want a larger number than 4096 (which seems high to me), then feel free to adjust it.